packages/block-library/src/post-comments-form/edit.js

```diff
@@ -66,6 +69,16 @@ export default function PostCommentsFormEdit( {
 				),
 				postType
 			);
+			const enableComments = () => setCommentStatus( 'open' );
+			actions = [
+				<Button
+					key="enableComments"
+					onClick={ enableComments }
+					variant="primary"
+				>
+					{ __( 'Enable comments' ) }
```

Comment:
Could use a `translators` note, IMO.

Reply:
What kind of information is most needed here?

Something like

```suggestion
					{ _x( 'Enable comments', 'button label' ) }
```

or more like

```suggestion
					{ _x( 'Enable comments', 'action that affects the current post' ) }
```

? 😄

Reply:
More the latter. :)

Reply:
d72be81